### PR TITLE
ACTI Indicator Query Playbook addition

### PR DESCRIPTION
**Pull request by satyakidroid**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
This release will focus on playbooks and some bug fixes.

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by kgal-pan** [comment]
@satyakidroid - Please fix the `reputation-ACTI_Intelligence_Alert.json` Indicator type by renaming it to `indicatortype-ACTI_Intelligence_Alert.json`. 


```bash
mv Packs/AccentureCTI/IndicatorTypes/reputation-ACTI_Intelligence_Alert.json Packs/AccentureCTI/IndicatorTypes/indicatortype-ACTI_Intelligence_Alert.json

demisto-sdk format -i Packs/AccentureCTI/IndicatorTypes/indicatortype-ACTI_Intelligence_Alert.json
```
Looks like it's the cause of [this build failure](https://app.circleci.com/pipelines/github/demisto/content/154267/workflows/07b505bd-4890-4739-bd88-247e3990ed91/jobs/500596).

**Comment by satyakidroid** [comment]
@kgal-pan  Do you know of this error?
image

**Comment by kgal-pan** [comment]
> @kgal-pan Do you know of this error? image

@satyakidroid - Yep, this means that your tasks in the Playbook have the `using` argument. This was used in 3 tasks (#1, #2, #3) in Playbook ` ACTI Indicator Enrichment`, see the example below for task #3:

image

Please remove those and replace them with `using-brand=AccentureCTI".

**Comment by satyakidroid** [comment]
> > @kgal-pan Do you know of this error? image
> 
> @satyakidroid - Yep, this means that your tasks in the Playbook have the `using` argument. This was used in 3 tasks (#1, #2, #3) in Playbook ` ACTI Indicator Enrichment`, see the example below for task #3:
> 
> image
> 
> Please remove those and replace them with `using-brand=AccentureCTI`.

Oh , yes

**Comment by kgal-pan** [comment]
@satyakidroid - Feel free to use the following reports to show where the UTs are missing coverage:

- [Vulnerability Query](https://output.circle-artifacts.com/output/job/3f12eedd-ce69-42aa-8e64-8ac66878e20c/artifacts/0/artifacts/coverage_report/html/d_c1f9202161de7c2e_ACTIVulnerabilityQuery_py.html)
- [Indicator Query](https://output.circle-artifacts.com/output/job/3f12eedd-ce69-42aa-8e64-8ac66878e20c/artifacts/0/artifacts/coverage_report/html/d_f7c4711416ac4d8a_ACTIIndicatorQuery_py.html)
- [Indicator Feed](https://output.circle-artifacts.com/output/job/3f12eedd-ce69-42aa-8e64-8ac66878e20c/artifacts/0/artifacts/coverage_report/html/d_681a8a3b1c868458_ACTIIndicatorFeed_py.html)

**Comment by satyakidroid** [comment]
@kgal-pan  yes, we are currently looking into these detailed reports

**Comment by satyakidroid** [comment]
@kgal-pan @yaakovi we have finished updating codebase to code standards.

**Comment by satyakidroid** [comment]
Hello @tomer-pan @yaakovi , We have completed making necessary changes

**Comment by satyakidroid** [comment]
@yaakovi  Hi, can you please approve this PR

File: Packs/AccentureCTI/Integrations/ACTIIndicatorQuery/ACTIIndicatorQuery.py [hunk 1]
```diff
@@ -364,12 +368,10 @@ def _enrich_analysis_result_with_intelligence(analysis_info, doc_search_client,
 
     alerts, reports = _get_ia_for_indicator(indicator, doc_search_client)
 
-    if alerts is not None:
-        analysis_info['Intelligence Alerts'] = alerts if len(
-            alerts) > 0 else 'No Intelligence Alert has been linked to this indicator'
-    if reports is not None:
-        analysis_info['Intelligence Reports'] = reports if len(
-            reports) > 0 else 'No Intelligence Report has been linked to this indicator'
+    if (alerts is not None) and (len(alerts) > 0):
```
Comment (yaakovi) [inline Packs/AccentureCTI/Integrations/ACTIIndicatorQuery/ACTIIndicatorQuery.py:371]:
```suggestion
    if alerts:
```

File: Packs/AccentureCTI/Integrations/ACTIIndicatorQuery/ACTIIndicatorQuery.py [hunk 2]
```diff
@@ -364,12 +368,10 @@ def _enrich_analysis_result_with_intelligence(analysis_info, doc_search_client,
...
+        analysis_info['Intelligence Alerts'] = alerts
+    if reports is not None and (len(reports) > 0):
```
Comment (yaakovi) [inline Packs/AccentureCTI/Integrations/ACTIIndicatorQuery/ACTIIndicatorQuery.py:373]:
```suggestion
    if reports:
```

**Review by yaakovi — APPROVED** [review]

**Comment by satyakidroid** [comment]
@yaakovi done making changes.

**Review by yaakovi — APPROVED** [review]

**Pull request merged**